config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1832,6 +1832,7 @@ presubmits:
         - --parallelism=1
         - --focus-regex=\[Serial\]
         - --skip-regex=\[Flaky\]|\[Benchmark\]|\[Feature:Eviction\]|\[Feature:CPUManager\]|\[Feature:MemoryManager\]|\[Feature:TopologyManager\]|\[Feature:NodeSwap\]|\[Feature:DynamicResourceAllocation\]|\[Feature:HugePages\]|\[Feature:PodLevelResources\]
+        - --system-spec-name=crio-cgroupv1
```

Comment:
This is just for the presubmit?

Should we also apply this change to the correspond periodic?